design-system/src/theme-provider.tsx

```diff
@@ -67,6 +67,7 @@ type ThemeProviderProps = {
 };
 
 const ThemeProvider = (props: ThemeProviderProps) => {
+  const themeName = props.theme || 'default'; // Use 'default' theme if none is provided
```

Comment:
Could we `return elementType;` since we don't have default font size for all of them?
And for the tone `return designTokens.colorPrimary;`?

Reply:
We still need to undo this changes.

Reply:
Yes, true, thanks for pointing that out. So I removed them from the `theme-provider.tsx `file - [here](https://github.com/commercetools/ui-kit/pull/2317/commits/b59c1a3bd960fa9389650ce076a0e2578433406d).
It seems that the other two `contexts.js` and `theme-context.js` files are alright, because they are the same as what is on the main branch. Let me know if there's anything else that I'm missing out.